lib/ts/recipe/openid/recipeImplementation.ts

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import { DiscoveryConfiguration, RecipeInterface, TypeNormalisedInput } from "./types";
+
+export default function getRecipeInterface(config: TypeNormalisedInput): RecipeInterface {
+    return {
+        getDiscoveryConfiguration: async function (): Promise<DiscoveryConfiguration> {
+            let issuer = config.issuerDomain.getAsStringDangerous() + config.issuerPath.getAsStringDangerous();
+            return {
+                status: "OK",
+                issuer,
+                jwks_uri: issuer + "/jwt/jwks.json",
```

Comment:
- Use the constant from the JWT Recipe here instead of hard coding "/jwt/jwks.json"

```
config.issuerDomain.getAsStringDangerous() + (config.issuerPath.append(new NormalisedURLPath("/jwt/jwks.json")).getAsStringDangerous())
```

Reply:
Done